src/components/Input/index.tsx

```diff
@@ -1,26 +1,88 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import classNames from 'classnames';
 
-import SimpleInput from '@/components/Input/SimpleInput';
+import styles from '@/components/Input/Input.css';
 
-type Category = 'simple';
 type Size = 'large' | 'medium';
+type State = '' | 'valid' | 'invalid';
 
 interface Props {
-  category?: Category;
+  className: string;
```

Comment:
this component should allow only the `variablesClassName`. What's the idea of having the `className` prop?

PS.: to modify the input look & feel, the dev needs to use the `variablesClassName` and pass the new values for the variables